src/components/templates/ScreeningRoom/useScreeningRoom.ts

```diff
@@ -119,24 +120,21 @@ export const useScreeningRoom = (screeningRoomVenueId: string) => {
   );
 
   const searchedVideos = useMemo(() => {
-    if (!searchQuery)
-      return filteredVideosBySubCategory.slice(0, displayedVideosAmount);
+    if (!searchQuery) return filteredVideosBySubCategory;
 
     return fuseVideos
       .search(searchQuery)
-      .slice(0, displayedVideosAmount)
       .map((fuseSearchItem) => fuseSearchItem.item);
-  }, [
-    searchQuery,
-    fuseVideos,
-    filteredVideosBySubCategory,
-    displayedVideosAmount,
-  ]);
+  }, [searchQuery, fuseVideos, filteredVideosBySubCategory]);
+
+  const displayedVideos = searchedVideos
+    .sort((a, b) => alphabeticalOrderSort(a.title, b.title))
```

Comment:
can we use `a.title.localeCompare(b.title)` here?

see: https://stackoverflow.com/a/2167619

Reply:
@sunny-viktoryia good catch!

Fixed in https://github.com/sparkletown/sparkle/pull/1464/commits/b406c60efbda905b3a7f0fca95cb9633c4496fab